ui/lib/utils.ts

```diff
@@ -177,3 +178,20 @@ export function getAppVersion(
     versionHref,
   };
 }
+
+// formatLogTimestamp formats a timestamp string in the RFC3339 format
+// to a human-readable format with UTC offset.
+// If the timestamp is undefined or an empty string, it returns "-".
+export function formatLogTimestamp(timestamp?: string, zone?: string): string {
+  if (!timestamp) {
+    return "-";
+  }
+
+  let dt = DateTime.fromISO(timestamp);
+
+  if (zone) {
+    dt = dt.setZone(zone);
+  }
+
+  return `${dt.toFormat("yyyy-LL-dd HH:MM:ss ZZ")} UTC`;
```

Comment:
I want to point out that printing the offset **and** "UTC" might come across as confusing. Usually you either print the offset or the **actual** timezone of the time displayed. [Here](https://pkg.go.dev/time#pkg-constants) is a list of common time/date patterns, none of which suggest printing both, the offset and the timezone, especially not a timezone that's different from the offset.

Suggestion:

```suggestion
  return `${dt.toFormat("yyyy-LL-dd HH:MM:ss ZZ")}`;
```

Reply:
Those patterns look somewhat unusual and not exhaustive, at least I don't think it's what is usually used on the frontend as human-readable formatted time. (Actually, I would not trust a list in Golang docs as a good example of time formatted for friendly user experience.)

In the formatted string +05:00 UTC was, strictly speaking, supposed to specify an offset, I can rename the param zone to offset or format it as "UTC+5" or "UTC+05:00", what is I think a more standard way of specifying offset.

In Figma the design looked like that:

<img width="285" alt="Screenshot 2023-02-02 at 23 54 04" src="https://user-images.githubusercontent.com/8824708/216589696-008bc676-9bc4-4584-99af-06af14d29bb6.png">

But we agreed with Josefa, we want local time instead of time in UTC time zone. So, it's either specifying offset (with or without UTC) or specifying a local timezone (like time with PST, for example, which I would need to check how to implement with Luxon).

I can ask Josefa what she would prefer.

Reply:
Wrote to Josefa about preferred time format. A useful link from Luxon docs on timezones:
https://github.com/moment/luxon/blob/master/docs/zones.md

Reply:
No, in Luxon API name for passed "UTC+3" (whatever) string is zone, so it's a fixed-offset zone and the param name will have to stay zone.